__tests__/server.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../data/dbConfig");
+const request = require("supertest");
+const server = require("../api/server");
+
+afterEach(async () => {
+  await db("games").truncate();
+});
+
+describe("the route handler", () => {
+  describe("get /games", () => {
+    it("responds with 200", async () => {
+      const response = await request(server).get("/games");
+      expect(response.status).toBe(200);
+    });
+    it("responds with json", async () => {
+      const response = await request(server).get("/games");
+      expect(response.type).toMatch(/json/i);
+    });
+    it("sends the correct response object", async () => {
```

Comment:
This test description could more closely follow the spec's description: 
`Write a test to make sure this endpoint always returns an array, even if there are no games stored. If there are no games to return, the endpoint should return an empty array`
It's testing the correct scenario, it could just be more apparent that we _always_ want that empty array if there are no games stored 👍